src/python/zquantum/qcbm/cost_function_test.py

```diff
@@ -65,44 +74,18 @@ def test_init(self):
 
     def test_evaluate(self):
         # Given
-        num_qubits = 4
-        topology = "all"
         epsilon = 1e-6
-        n_layers = 1
-        params = np.array([0, 0, 0, 0])
-        distance_measure = compute_clipped_negative_log_likelihood
-        ansatz = get_qcbm_ansatz(num_qubits, n_layers, topology)
-        target_bitstring_distribution = BitstringDistribution(
```

Comment:
Just curious – why did you get rid of all this?

Reply:
It's now declared in the `setUp` for the unittest class